topo_processor/metadata/metadata_loaders/metadata_loader_imagery_historic.py

```diff
@@ -109,6 +106,31 @@ def read_csv(self, metadata_file: str = "") -> None:
 
         self.is_init = True
 
+    def add_centroid(self, item: Item, asset_metadata: Dict[str, str]):
+        centroid = {
+            "lat": string_to_number(asset_metadata.get("photocentre_lat", None)),
+            "lon": string_to_number(asset_metadata.get("photocentre_lon", None)),
+        }
+        if not all(centroid.values()) or "" in centroid.values():
+            item.add_warning(
+                msg="Skipped Record",
+                cause=self.name,
+                e=Exception(f"stac field 'proj:centroid' has None values: {centroid}"),
+            )
+        else:
+            item.properties.update({"proj:centroid": centroid})
```

Comment:
I think its clearer to do `item.properties['proj:centorid'] = centorid` than doing a `.update` and creating a whole new dict for it.